src/auth/basic/SSPI/basic_sspi_auth.cc

```diff
@@ -149,7 +151,23 @@ main(int argc, char **argv)
         /* Clear any current settings */
         username[0] = '\0';
         password[0] = '\0';
-        sscanf(wstr, "%s %s", username, password);  /* Extract parameters */
+
+        parsed = sscanf(wstr, " %255s%1c%255s %c", username, &sep, password, &junk);
+
+        if (parsed < 2 || !isspace((unsigned char)sep)) {
```

Comment:
Can we simplify like this (and remove other `parsed` checks)?


```suggestion
        if (parsed != 3 || !isspace(sep)) {
```

Reply:
Any reason to not include compat/xis.h and use xisspace ?